src/Microsoft.Identity.Web/AzureFunctionsAuthenticationHttpContextExtension.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Extensions for <see cref="AzureFunctionsAuthenticationHttpContextExtension"/>.
+    /// </summary>
+    public static class AzureFunctionsAuthenticationHttpContextExtension
+    {
+        /// <summary>
+        /// Enables Bearer authentication for an API for use in Azure Functions.
+        /// </summary>
+        /// <param name="httpContext">The current HTTP Context, such as req.HttpContext.</param>
+        /// <returns>A task indicating success or failure. In case of failure <see cref="Microsoft.Identity.Web.UnauthorizedObjectResult"/>.</returns>
+        public static async Task<(bool, IActionResult?)> AuthenticateAzureFunctionAsync(
+            this HttpContext httpContext)
+        {
+            if (httpContext == null)
+            {
+                throw new ArgumentNullException("Parameter httpContext cannot be null");
+            }
+
+            AuthenticateResult? result =
+                await httpContext.AuthenticateAsync("Bearer").ConfigureAwait(false);
```

Comment:
>"Bearer" [](start = 52, length = 8)

we have a constant value-> `Constants.Bearer`

Reply:
Addressed